### PR TITLE
Enable ports 5900-5999 for HTML5 console

### PR DESCRIPTION
**Pull request by carbonin**
Removed the `firewall` command in favor of opening
ports in the post section.

The kickstart `firewall` command was adding the ports
to the default zone which is "public" at the time the
main section runs.

We later create the "manageiq" zone and set that to the
default causing the ports to still be closed in the active
zone.

https://bugzilla.redhat.com/show_bug.cgi?id=1278432

### CODE REVIEW

**Comment by carbonin** [comment]
@abellotti @gtanzillo @martinpovolny

**Comment by gtanzillo** [comment]
:+1: LGTM

File: kickstarts/partials/post/firewalld.ks.erb [hunk 1]
```diff
@@ -15,3 +15,10 @@ firewall-offline-cmd --zone=manageiq --add-service=http
 firewall-offline-cmd --zone=manageiq --add-service=https
 firewall-offline-cmd --zone=manageiq --add-service=ssh
 firewall-offline-cmd --zone=manageiq --add-service=postgresql
+firewall-offline-cmd --zone=manageiq --add-port=22/tcp
+firewall-offline-cmd --zone=manageiq --add-port=80/tcp
+firewall-offline-cmd --zone=manageiq --add-port=80/udp
+firewall-offline-cmd --zone=manageiq --add-port=443/tcp
+firewall-offline-cmd --zone=manageiq --add-port=443/udp
+firewall-offline-cmd --zone=manageiq --add-port=5432/tcp
+firewall-offline-cmd --zone=manageiq --add-port=5900-5999/tcp
```
Comment (kbrock) [inline kickstarts/partials/post/firewalld.ks.erb]: this is so much nicer than listing out every port.

Comment (carbonin) [inline kickstarts/partials/post/firewalld.ks.erb]: And it works :smile:

**Pull request merged**